var/spack/repos/builtin/packages/trilinos/package.py

```diff
@@ -113,7 +116,7 @@ def url_for_version(self, version):
     depends_on('py-numpy', when='+python')
     depends_on('swig', when='+python')
 
-    patch('umfpack_from_suitesparse.patch')
+    patch('umfpack_from_suitesparse.patch', when='@:12.8.1')
```

Comment:
@BarrySmith please rebase on top of the current `develop`. New Trilinos has been added and for which the patch is no longer needed.